EAP-TLS_Solution/EAP-TLS Client/main.c

```diff
@@ -0,0 +1,254 @@
+﻿/***************************************************************************************
+	This is an EAP-TLS Client reference solution for Azure Sphere-based devices.
```

Comment:
Should we comb the comments and replace "reference solution" with just "solution"?